pkg/workerapi/workerclient/grpcclient.go

```diff
@@ -0,0 +1,57 @@
+package workerclient
+
+import (
+	"errors"
+	"fmt"
+
+	v1 "github.com/iver-wharf/wharf-cmd/api/workerapi/v1"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+type grpcClient struct {
+	address string
+	opts    ClientOptions
+
+	client v1.WorkerClient
+	conn   *grpc.ClientConn
+}
+
+var errAlreadyOpen = errors.New("client is already open")
+
+// NewRPCClient creates a new gRPC Client that can communicate with the Worker
+// gRPC server.
+func newGRPCClient(address string, opts ClientOptions) *grpcClient {
+	return &grpcClient{
+		address: address,
+		opts:    opts,
+	}
+}
+
+func (c *grpcClient) open() error {
+	if c.conn != nil {
+		return errAlreadyOpen
+	}
```

Comment:
nit: Suggest just returning nil here instead and removing the `errAlreadyOpen` variable. Also suggest renaming the method to something like `ensureOpen` or `ensureConnected`, so it's still clear that it can be called when already connected.